src/EnergyPlus/CurveManager.cc

```diff
@@ -120,16 +120,14 @@ namespace CurveManager {
     void BtwxtMessageCallback(const Btwxt::MsgLevel messageType, const std::string message, void *contextPtr)
     {
         std::pair<EnergyPlusData *, std::string> contextPair = *(std::pair<EnergyPlusData *, std::string> *)contextPtr;
-        std::string fullMessage = contextPair.second + ": " + message;
+        std::string fullMessage = format("{}: {}", contextPair.second, message);
         if (messageType == Btwxt::MsgLevel::MSG_ERR) {
```

Comment:
I have many things to say about Btwxt, but I will refrain in this PR.

Reply:
@amirroth, I'm all ears! We're working on cleaning up the error handling now, but I suspect we can improve some of the calculations too.

Reply:
:+1:

Reply:
I won't do it in this PR of course. Maybe slack.